### PR TITLE
Model validation

### PR DESCRIPTION
**Pull request by Olesia-Dev**

### CODE REVIEW

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java [hunk 1]
```diff
@@ -2,14 +2,27 @@
...
+
+    @NotBlank
+    @Size(max = 64)
     private String lastName;
+
+    @Email
     private String email;
+
+    @Size(max = 256)
     private String currentPosition;
```
Comment (LypchanskyiYurii) [inline src/main/java/com/zaiats/catcher/model/CandidateModel.java:26]: Where is @Size 
You need to add a value min (2-3 letters).

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java [hunk 2]
```diff
@@ -2,14 +2,27 @@
...
+import javax.validation.constraints.Size;
 import java.util.Objects;
 
 public class CandidateModel {
 
     private Long id;
+
+    @NotBlank
+    @Size(max = 64)
     private String firstName;
```
Comment (LypchanskyiYurii) [inline src/main/java/com/zaiats/catcher/model/CandidateModel.java]: I think it was better to add the value min (2-3 letters)

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java [hunk 3]
```diff
@@ -2,14 +2,27 @@
...
+
+    @NotBlank
+    @Size(max = 64)
     private String lastName;
```
Comment (LypchanskyiYurii) [inline src/main/java/com/zaiats/catcher/model/CandidateModel.java]: 1. I would also like to add a minimum value here.
2. Add message, when request will empty return message.
`@NotBlank(message = "{name.not.empty}")`

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java [hunk 4]
```diff
@@ -2,14 +2,27 @@
...
+
+    @Email
     private String email;
```
Comment (LypchanskyiYurii) [inline src/main/java/com/zaiats/catcher/model/CandidateModel.java]: 1. Add message, when email will not valid return message.
`@Email(message = "{email.not.valid}")`

2. Add annotation   
@NotBlank

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java (hunk 2 above)
Comment (Olesia-Dev) [reply src/main/java/com/zaiats/catcher/model/CandidateModel.java:16]: It should have at least 1 character according to the pair of those validation annotations.

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java (hunk 3 above)
Comment (Olesia-Dev) [reply src/main/java/com/zaiats/catcher/model/CandidateModel.java:20]: 1. Same as prev.
2. Default validation message is pretty self-descriptive here.

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java (hunk 2 above)
Comment (Olesia-Dev) [reply src/main/java/com/zaiats/catcher/model/CandidateModel.java]: Both annotations in pair means 1-64 characters.

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java (hunk 3 above)
Comment (Olesia-Dev) [reply src/main/java/com/zaiats/catcher/model/CandidateModel.java]: 1. The same as previous.
2. Added specific messages to all validations.

File: src/main/java/com/zaiats/catcher/model/CandidateModel.java (hunk 4 above)
Comment (Olesia-Dev) [reply src/main/java/com/zaiats/catcher/model/CandidateModel.java]: 1. Done!
2. Good catch ^^)
Done.

**Review by LypchanskyiYurii — APPROVED** [review]

**Pull request merged**